tendrl/AlertNotificationManager.py

```diff
@@ -0,0 +1,28 @@
+import tendrl.utils as utils
+import multiprocessing
+from tendrl.MailHandler import MailHandler
```

Comment:
source names you could keep all lowercase os it would become something like

from tendrl.mail_handler import MailHandler (class names could be like as you have)